LPS-177564 Navigate to FDS Views page on the 'view' action of FDS Entry

markocikos opened the pull request:
### References

- [LPS-177564 Navigate to FDS Views page on the 'view' action of FDS Entry](https://issues.liferay.com/browse/LPS-177564)
- [Figma mockup](https://www.figma.com/file/VTOLTBFe99y9kA7AY3TcNR/lps-175276-design-app-to-manage-datasets?node-id=937-117695&t=gNfcz6mQplE92gsc-0)

### What is the goal of this PR?

This is the first PR of the story to manage FDS Views. We are adding navigation from FDS Entry action to the placeholder of FDS Views page.

Notes:
- The changes in FDS Views Manager are minimal, basically a sample of frontend setup of actions, with a placeholder Views page. I'll send FDS Views persistence and CRUD handling in a separate PR.
- Setting of `onClick` in actions was bugged, this is a part of our discovery of issues when using FDS directly as a React component. In the PR we fixing this, but also taking the opportunity to massively simplify handling of actions in FDS. @carloslancha I had something like this in mind when we discussed exposing of FDS context in custom actions. There is now a single point of placing exposed values, in `exposedProps` const. There are for sure other ways to simplify code, but I had to stop myself :slightly_smiling_face: 
  - As a part of the changes, we are fixing accessibility `aria-label` setting, fixing `PropTypes` setting, and removing unused `size` setting.
  - It may be that I missed some edge case, but I did test several places in commerce and FDS sample module.
- Inline icon for action is temporary, until we add other actions. Then, it will become a dropdown, as in mockups. cc @ugeortiz

### What does it look like?

https://user-images.githubusercontent.com/5435511/223713113-aa76822e-c2b1-4cca-a13b-d9cebc59d7a6.mp4


### Steps to reproduce
1. Add `feature.flag.LPS-164563=true` to `portal-ext.properties`
2. Open Global Menu > Control Panel > Object > Datasets

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-32.liferay.com/job/test-portal-source-format/5718/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/5718/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 6 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7167fb45ee3ac3dc7e131e36c7a8d06997a43cf3">7167fb45ee3ac3dc7e131e36c7a8d06997a43cf3</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/markocikos/liferay-portal/tree/LPS-177564">LPS-177564</a><br/>Branch GIT ID: <a href="https://github.com/markocikos/liferay-portal/commit/39b2bc559f1490baa29f9636ee8cd68c8f16aedb">39b2bc559f1490baa29f9636ee8cd68c8f16aedb</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-source-format/5718/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/5718/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-32.liferay.com/job/test-portal-source-format/5718/">test-portal-source-format#5718</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/5718/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/3103">liferay-frontend#3103</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2224765190">[master] ci:test:sf - markocikos &gt; liferay-frontend - PR#3103 - 2023-03-08[04:47:29]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-32.liferay.com/job/publish-testray-report/20748/">publish-testray-report#20748</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/5968/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5968/jenkins-report.html">here</a>.</b></p>

kresimir-coko commented on modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.js:
```diff
@@ -190,47 +96,103 @@ function Actions({actions, itemData, itemId, menuActive, onMenuActiveChange}) {
+			status,
+			successMessage,
+			title,
+		} = data ?? {};
```
Neat trick! I wonder how the rest of the code functions if `data` is falsy, do we need any sanity checks?

markocikos replied:
There is one other place where `data` is used, and we do have sanity check:
 ```js
executeAsyncItemAction({
	...
	method: method ?? data?.method,
	...
});
```
This is one the things that can be simplified, there is no point in `method` being defined in two places in API. But, I'd put that outside of scope of this PR. We would need to check all usages, possibly change some of them.

kresimir-coko commented on modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.js:
```diff
@@ -190,47 +96,103 @@ function Actions({actions, itemData, itemId, menuActive, onMenuActiveChange}) {
+			else if (target === 'sidePanel') {
+				event.preventDefault();
+
+				highlightItems([itemId]);
+				openSidePanel({
```
Empty line or am I trolling?

markocikos replied:
/giphy they see me trolling

kresimir-coko commented on modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/ActionsDropdown.js:
```diff
@@ -88,11 +75,10 @@ function ActionsDropdown({
 	const isMounted = useIsMounted();
 
-	const editModeActive = !!frontendDataSetContext.itemsChanges[itemId];
+	const editModeActive = !!itemsChanges[itemId];
 	const itemChanges =
```
Probably empty line missing. I remember the rule empty line between declaration and usage

markocikos replied:
That's a rule in Java, currently optional in JS.

kresimir-coko reviewed (changes requested):
Small things but looking great

markocikos commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx:
```diff
@@ -430,9 +432,25 @@ const FDSEntries = ({
 		],
 	};
 
+	const onViewClick = ({itemData}: any) => {
```
Oops! That should have been temporary, to get TS out of the way until logic is done :sweat_smile:

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 28 out of 28 jobs passed</h3><h3>:x: ci:test:relevant - 73 out of 77 jobs passed in 3 hours 30 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4364">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1287401_5968</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7167fb45ee3ac3dc7e131e36c7a8d06997a43cf3">7167fb45ee3ac3dc7e131e36c7a8d06997a43cf3</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/e102c80faa71cd74b3e24a55832c0018da6fbd83">e102c80faa71cd74b3e24a55832c0018da6fbd83</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2224548361">EE Development Acceptance (master) - 4363 - 2023-03-07[20:15:46]</a></p><details><summary><strong>ci:test:stable - 28 out of 28 jobs PASSED</strong></summary><details><summary><strong>28 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 73 out of 77 jobs PASSED</strong></summary><div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/5968/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>build-iw-jdk8<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/168284/">build-iw-jdk8/0/0</a></li></ul></li><li>build-lib-versions-jdk8<ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/178418/">build-lib-versions-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/417618/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-jdk8<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/168286/">empty-osgi-core-dir-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-db2115-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/443567/">functional-smoke-tomcat90-db2115-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-hypersonic20-jdk8<ul><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/139431/">functional-smoke-tomcat90-hypersonic20-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/417615/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/4266/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-tomcat90-oracle193-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/507697/">functional-smoke-tomcat90-oracle193-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/173721/">functional-smoke-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-sqlserver2019-jdk8<ul><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/152991/">functional-smoke-tomcat90-sqlserver2019-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/485352/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/171752/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-chrome100-jdk8<ul><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179397/">functional-tomcat90-mysql57-chrome100-jdk8/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/507695/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/1</strike></a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/507696/">functional-tomcat90-mysql57-chrome100-jdk8/0/2</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/419007/">functional-tomcat90-mysql57-chrome100-jdk8/0/3</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/180203/">functional-tomcat90-mysql57-chrome100-jdk8/0/4</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/417617/">functional-tomcat90-mysql57-chrome100-jdk8/0/5</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/413631/">functional-tomcat90-mysql57-chrome100-jdk8/0/6</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/4267/">functional-tomcat90-mysql57-chrome100-jdk8/0/7</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/413632/">functional-tomcat90-mysql57-chrome100-jdk8/0/8</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461121/">functional-tomcat90-mysql57-chrome100-jdk8/0/9</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/175527/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/10</strike></a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/173407/">functional-tomcat90-mysql57-chrome100-jdk8/0/11</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/152989/">functional-tomcat90-mysql57-chrome100-jdk8/0/12</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/180202/">functional-tomcat90-mysql57-chrome100-jdk8/0/13</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/152990/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/14</strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/170716/">functional-tomcat90-mysql57-chrome100-jdk8/0/15</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/152444/">functional-tomcat90-mysql57-chrome100-jdk8/0/16</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/161670/">functional-tomcat90-mysql57-chrome100-jdk8/0/17</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/164358/">functional-tomcat90-mysql57-chrome100-jdk8/0/18</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/165066/">functional-tomcat90-mysql57-chrome100-jdk8/0/19</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/475991/">functional-tomcat90-mysql57-chrome100-jdk8/0/20</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/417616/">functional-tomcat90-mysql57-chrome100-jdk8/0/21</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/475990/">functional-tomcat90-mysql57-chrome100-jdk8/0/22</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/411001/">functional-tomcat90-mysql57-chrome100-jdk8/0/23</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/170717/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/485354/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/485353/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388515/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/419006/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/284142/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>lpkg-controller-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/419005/">lpkg-controller-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/160145/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/410999/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261713/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/167121/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/411000/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/417614/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461120/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388513/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/443566/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/168285/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/485351/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/413629/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/164359/">modules-compile-jdk8/0/12</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/419009/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/443569/">modules-integration-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/443568/">modules-integration-mysql57-jdk8/0/2</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/507698/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/419008/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/428066/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388514/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/413630/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/411002/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261712/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/139430/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-jdk8<ul><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/175528/">patching-tool-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/165067/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/419004/">portal-license-smoke-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/386531/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/173723/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/173722/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/386530/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/176347/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>73 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5968/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/175527//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/10</a><ol><li><div><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/175527//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_FDSQuickActions_CanExecuteAnAction_">LocalFile.FDSQuickActions#CanExecuteAnAction</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-28/test-portal-acceptance-pullrequest(master)/5968/functional-tomcat90-mysql57-chrome100-jdk8/0/10/LocalFile.FDSQuickActions_CanExecuteAnAction/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-28/test-portal-acceptance-pullrequest(master)/5968/functional-tomcat90-mysql57-chrome100-jdk8/0/10/LocalFile.FDSQuickActions_CanExecuteAnAction/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-28/test-portal-acceptance-pullrequest(master)/5968/functional-tomcat90-mysql57-chrome100-jdk8/0/10/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2224566212/history">Failed 0 of last 24</a></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at e102c80faa71cd74b3e24a55832c0018da6fbd83:</strong></summary><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/507695//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/1</a><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/507695//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_UsersAdmin_EditUserSite_">LocalFile.UsersAdmin#EditUserSite</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-28/test-portal-acceptance-pullrequest(master)/5968/functional-tomcat90-mysql57-chrome100-jdk8/0/1/LocalFile.UsersAdmin_EditUserSite/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-28/test-portal-acceptance-pullrequest(master)/5968/functional-tomcat90-mysql57-chrome100-jdk8/0/1/LocalFile.UsersAdmin_EditUserSite/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-28/test-portal-acceptance-pullrequest(master)/5968/functional-tomcat90-mysql57-chrome100-jdk8/0/1/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2224563505/history">Failed 7 of last 24</a></div></li></ol></div></li><li><div><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/152990//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/14</a><ol><li><div><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/152990//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_FrontendDataSetAdminSearch_CanReturnNoMatchedResults_">LocalFile.FrontendDataSetAdminSearch#CanReturnNoMatchedResults</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-28/test-portal-acceptance-pullrequest(master)/5968/functional-tomcat90-mysql57-chrome100-jdk8/0/14/LocalFile.FrontendDataSetAdminSearch_CanReturnNoMatchedResults/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-28/test-portal-acceptance-pullrequest(master)/5968/functional-tomcat90-mysql57-chrome100-jdk8/0/14/LocalFile.FrontendDataSetAdminSearch_CanReturnNoMatchedResults/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-28/test-portal-acceptance-pullrequest(master)/5968/functional-tomcat90-mysql57-chrome100-jdk8/0/14/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2224566594/history">Failed 10 of last 10</a></div></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-28/jobs/test-portal-acceptance-pullrequest(master)/builds/5968/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-28/jobs/test-portal-acceptance-pullrequest(master)/builds/5968/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-28/jobs/test-portal-acceptance-pullrequest(master)/builds/5968/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest(master)/5968/">test-portal-acceptance-pullrequest(master)#5968</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5968/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/3103">liferay-frontend#3103</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2224962074">[master] ci:test:relevant - markocikos &gt; liferay-frontend - PR#3103 - 2023-03-08[04:55:31]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-28.liferay.com/job/publish-testray-report/14033/">publish-testray-report#14033</a><br/></div></div>

markocikos commented:
@kresimir-coko Force-pushed! Changes:
- Fixed `dataType`, based on https://github.com/liferay-frontend/liferay-portal/pull/3103#discussion_r1129575381
- SFd, based on trolling
- Moved `itemsActions` definition in its right place, as we discussed in Slack
- Fixed unconditional prevention of default, that caused CI error in https://github.com/liferay-frontend/liferay-portal/pull/3103#issuecomment-1460455812

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/474/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/474/jenkins-report.html">here</a>.</b></p>

kresimir-coko commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx:
```diff
@@ -463,6 +472,13 @@ const FDSEntries = ({
 					provider: ProviderRenderer,
 				}}
 				id={`${namespace}FDSEntries`}
+				itemsActions={[
+					{
+						icon: 'view',
+						label: Liferay.Language.get('view'),
+						onClick: onViewClick,
+					},
+				]}
```
I see you decided to leave it here in `views`.

kresimir-coko commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx:
```diff
@@ -430,6 +430,15 @@ const FDSEntries = ({
 		],
 	};
 
+	const onViewClick = ({itemData}: {itemData: FDSEntry}) => {
```
Can't this just be `const onViewClick = ({itemData}: FDSEntry) => {` ?

markocikos replied:
That would mean that `itemData` is a property of `FDSEntry`. TS Error:
```ts
Property 'itemData' does not exist on type 'FDSEntry'.
```

kresimir-coko replied:
Ah, okay, still rusty with TS

kresimir-coko commented on modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/index.d.ts:
```diff
@@ -69,6 +69,7 @@ type TItemsActions = {
 	icon?: string;
 	id?: string;
 	label?: string;
+	onClick?: Function;
```
So you added `onClick` to `TItemsActions`, what do I do if I need a small modal as a `target`? Should I add the type as a part of my changes?

markocikos replied:
If you need it, yes. We don't need to be comprehensive about types, adding only used properties. It's quite likely that we have unused ones. We can make a comprehensive sweep when we migrate FDS to TS. 

On a sidenote, you should probably not use modal target, as this uses custom FDS modal. We intend to replace it with DXP standard `openModal`. In general, `action.target` is not a great API. The main reason for it is to have configuration in Java. You set a keyword, `target`, and this determines hardcoded frontend behavior. We don't need to limit ourselves in that way when using `FrontendDataSet` React component directly.

kresimir-coko replied:
Okay, so I guess something like an onClick that opens our frontend-js-web modal would be the preferred route?

markocikos replied:
yep :+1:

kresimir-coko reviewed (approved):
Tiny questions

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/474/">test-portal-acceptance-pullrequest(master)#474</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/474/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/3103">liferay-frontend#3103</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2226215466">[master] ci:test:relevant - markocikos &gt; liferay-frontend - PR#3103 - 2023-03-09[04:00:15]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/667/">publish-testray-report#667</a><br/></div></div>

markocikos commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/8511/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8511/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-2.liferay.com/job/test-portal-source-format/1390/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-2.liferay.com/userContent/jobs/test-portal-source-format/builds/1390/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 28 out of 28 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 75 out of 77 jobs passed in 1 hour 20 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7167fb45ee3ac3dc7e131e36c7a8d06997a43cf3">7167fb45ee3ac3dc7e131e36c7a8d06997a43cf3</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7167fb45ee3ac3dc7e131e36c7a8d06997a43cf3">7167fb45ee3ac3dc7e131e36c7a8d06997a43cf3</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2225028771">EE Development Acceptance (master) - 4364 - 2023-03-08[05:07:06]</a></p><details><summary><strong>ci:test:stable - 28 out of 28 jobs PASSED</strong></summary><details><summary><strong>28 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 74 out of 77 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/8511/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>build-iw-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/488579/">build-iw-jdk8/0/0</a></li></ul></li><li>build-lib-versions-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264919/">build-lib-versions-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388951/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/446325/">empty-osgi-core-dir-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-db2115-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/510771/">functional-smoke-tomcat90-db2115-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-hypersonic20-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/420827/">functional-smoke-tomcat90-hypersonic20-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/420826/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391145/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-tomcat90-oracle193-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/416289/">functional-smoke-tomcat90-oracle193-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7587/">functional-smoke-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-sqlserver2019-jdk8<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7589/">functional-smoke-tomcat90-sqlserver2019-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7591/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463972/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-chrome100-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388952/">functional-tomcat90-mysql57-chrome100-jdk8/0/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391147/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/1</strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/181947/">functional-tomcat90-mysql57-chrome100-jdk8/0/2</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388953/">functional-tomcat90-mysql57-chrome100-jdk8/0/3</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391148/">functional-tomcat90-mysql57-chrome100-jdk8/0/4</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/510772/">functional-tomcat90-mysql57-chrome100-jdk8/0/5</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264916/">functional-tomcat90-mysql57-chrome100-jdk8/0/6</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388948/">functional-tomcat90-mysql57-chrome100-jdk8/0/7</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388949/">functional-tomcat90-mysql57-chrome100-jdk8/0/8</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388950/">functional-tomcat90-mysql57-chrome100-jdk8/0/9</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463967/">functional-tomcat90-mysql57-chrome100-jdk8/0/10</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463968/">functional-tomcat90-mysql57-chrome100-jdk8/0/11</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7586/">functional-tomcat90-mysql57-chrome100-jdk8/0/12</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/488577/">functional-tomcat90-mysql57-chrome100-jdk8/0/13</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7588/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/14</strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/172680/">functional-tomcat90-mysql57-chrome100-jdk8/0/15</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/180274/">functional-tomcat90-mysql57-chrome100-jdk8/0/16</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463971/">functional-tomcat90-mysql57-chrome100-jdk8/0/17</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463970/">functional-tomcat90-mysql57-chrome100-jdk8/0/18</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/488578/">functional-tomcat90-mysql57-chrome100-jdk8/0/19</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/180276/">functional-tomcat90-mysql57-chrome100-jdk8/0/20</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391146/">functional-tomcat90-mysql57-chrome100-jdk8/0/21</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/446326/">functional-tomcat90-mysql57-chrome100-jdk8/0/22</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/181946/">functional-tomcat90-mysql57-chrome100-jdk8/0/23</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/488584/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/414017/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7594/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/420828/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/416287/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7593/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>lpkg-controller-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/446322/">lpkg-controller-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/488580/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/446323/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463973/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/172681/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463974/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/414015/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7592/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/510770/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/175378/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/446324/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7590/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/180275/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/510769/">modules-compile-jdk8/0/12</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388954/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264921/">modules-integration-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/416290/">modules-integration-mysql57-jdk8/0/2</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264920/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/420829/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/510773/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/488583/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/416288/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264917/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/414014/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/488581/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/414013/">patching-tool-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264918/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463969/">portal-license-smoke-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/175379/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/414016/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/488582/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463976/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/463975/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>74 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8511/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 7167fb45ee3ac3dc7e131e36c7a8d06997a43cf3:</strong></summary><ol><li><div><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391147//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/1</a><ol><li><div><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391147//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_UsersAdmin_EditUserSite_">LocalFile.UsersAdmin#EditUserSite</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-32/test-portal-acceptance-pullrequest(master)/8511/functional-tomcat90-mysql57-chrome100-jdk8/0/1/LocalFile.UsersAdmin_EditUserSite/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-32/test-portal-acceptance-pullrequest(master)/8511/functional-tomcat90-mysql57-chrome100-jdk8/0/1/LocalFile.UsersAdmin_EditUserSite/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-32/test-portal-acceptance-pullrequest(master)/8511/functional-tomcat90-mysql57-chrome100-jdk8/0/1/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2225029586/history">Failed 8 of last 24</a></div></li></ol></div></li><li><div><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7588//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/14</a><ol><li><div><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/7588//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_FrontendDataSetAdminSearch_CanReturnNoMatchedResults_">LocalFile.FrontendDataSetAdminSearch#CanReturnNoMatchedResults</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-32/test-portal-acceptance-pullrequest(master)/8511/functional-tomcat90-mysql57-chrome100-jdk8/0/14/LocalFile.FrontendDataSetAdminSearch_CanReturnNoMatchedResults/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-32/test-portal-acceptance-pullrequest(master)/8511/functional-tomcat90-mysql57-chrome100-jdk8/0/14/LocalFile.FrontendDataSetAdminSearch_CanReturnNoMatchedResults/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-03/test-1-32/test-portal-acceptance-pullrequest(master)/8511/functional-tomcat90-mysql57-chrome100-jdk8/0/14/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2225040505/history">Failed 11 of last 11</a></div></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-32/jobs/test-portal-acceptance-pullrequest(master)/builds/8511/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-32/jobs/test-portal-acceptance-pullrequest(master)/builds/8511/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-32/jobs/test-portal-acceptance-pullrequest(master)/builds/8511/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/8511//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/131418
[Console](https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/8511//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/8511/">test-portal-acceptance-pullrequest(master)#8511</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8511/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/3103">liferay-frontend#3103</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2226401524">[master] ci:test:relevant - markocikos &gt; liferay-frontend - PR#3103 - 2023-03-09[08:59:46]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-32.liferay.com/job/publish-testray-report/20775/">publish-testray-report#20775</a><br/></div></div>
